src/main/java/com/urswolfer/intellij/plugin/gerrit/git/GerritGitUtil.java

```diff
@@ -426,6 +427,18 @@ public GitLineHandler compute() {
         }
     }
 
+    public Optional<VcsFullCommitDetails> loadGitCommitInfo(Project project, GitRepository gitRepository, ChangeInfo changeInfo) throws VcsException {
+        List<GitCommit> history = GitHistoryUtils.history(project, gitRepository.getRoot(), changeInfo.id);
```

Comment:
`changeInfo.id` does not work in my case, it is not a git commit hash.

You can take a look at `gerritUtil.getFirstFetchInfo(changeDetails);`